tests/unit/lib/hotkeyManager.spec.ts

```diff
@@ -0,0 +1,127 @@
+import { it } from "vitest";
+import { HotkeyManager, HotkeysJs } from "@/plugins/hotkeyPlugin";
+
+type DummyHotkeysJs = HotkeysJs & {
+  registeredHotkeys: {
+    key: string;
+    scope: string;
+    // callbackを持たせると比較が面倒になるので持たせない
+    // callback: (e: KeyboardEvent) => void;
+  }[];
+  currentScope: string;
+};
+const createHotkeyManager = (): {
+  hotkeyManager: HotkeyManager;
+  dummyHotkeysJs: DummyHotkeysJs;
+} => {
+  const registeredHotkeys: DummyHotkeysJs["registeredHotkeys"] = [];
+  const dummyHotkeysJs = ((
+    key: string,
+    { scope }: { scope: string },
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    callback: (e: KeyboardEvent) => void
+  ) => {
+    registeredHotkeys.push({ key, scope });
+  }) as DummyHotkeysJs;
+  dummyHotkeysJs.unbind = (key: string) => {
+    const index = dummyHotkeysJs.registeredHotkeys.findIndex(
+      (h) => h.key === key
+    );
+    if (index !== -1) {
+      registeredHotkeys.splice(index, 1);
+    }
```

Comment:
こっちは未登録だった場合にエラーにすべきかも
```suggestion
    if (index !== -1) {
      registeredHotkeys.splice(index, 1);
    } else {
      throw new Error(`key: ${key} is not found`);
    }
```